custom_components/enpal/sensor.py

```diff
@@ -16,18 +16,17 @@
 from influxdb_client import InfluxDBClient
 
 _LOGGER = logging.getLogger(__name__)
-SCAN_INTERVAL = timedelta(seconds=120)
+SCAN_INTERVAL = timedelta(seconds=20)
 
 VERSION= '0.1.0'
```

Comment:
You've updated manifest to version 0.2.0, so why not this line?

Reply:
next mistake -.-

Reply:
fixed it in this branch iam sorry